src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -936,6 +938,8 @@ private PackageLicenseInfo GeneratePackageLicenseInfo(IPackageSearchMetadata met
             // Group actions by project
             var actionsByProject = expandedActions.GroupBy(action => action.ProjectId);
 
+            Dictionary<string, SortedSet<string>>? newSourceMappings = null;
```

Comment:
I assume we are using `SortedSet` type because we need the data in alphabetical order. Correct?

Reply:
I also needed a simple way to get uniqueness as projects are iterated for their new mapped packages. Imagine a huge solution where the same package is being installed across a dozen projects, with dozens of transitive dependencies each. I figured Set is appropriate, and chose SortedSet since I also need the final rendered list sorted.